File: lab4/lang-tests/test-2/typed.c
#include <stdio.h>
#include <stdlib.h>

int main(void){

    int num = 123;
    printf("\n'a' es una variable de tipo entero\n");
    
    char* word = "hola a todos";
    printf("palabra es un string: %s\n\n", word);
    
    int result;
    result = word + num;
    printf("el resultado es %d\n",result);
    printf("C ES DE TIPADO DEBIL\n");

    return 0;
}
// c es de tipado DEBIL

File: lab4/lang-tests/test-2/typed.py
"""
PYTHON es un lenguaje de tipado DEBIL
"""

def tipado_debil():
    num = 10
    print "num = ", num
    print "num es", type(num) 
    print "\n"
    
    word = "hola a todos\n"
    print "word = ", word
    print "word es", type(word)
    
    result = word * num
    print "result:"
    print result
    print "result es", type(result)
    print("Python es de tipado DEBIL\n")

tipado_debil()
